ming_sim/cli_backend.py

```
@@ -986,7 +997,14 @@ def classify_cli_action_intent(
         "- 不得因出现密查/查访字样就判密令；整体为问则无；"
         "整体为令时按是否指向现有密令选更新或新建。\n"
         "- 更新/催办/提交核议/记进展仅针对【现有密令】；无现有密令时不要硬判这四者"
-        "（新建不受此限）。非妃嫔不要硬判调教。\n\n"
+        "（新建不受此限）。非妃嫔不要硬判调教。\n"
+        "交办·责成承诺契约：\n"
+        f"- 当前回合={turn_n}。相对期限填期限月数=N（连续N月/三月内）；"
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)